.eslintignore

```diff
@@ -7,6 +7,8 @@
 /react-native/shared
 /shared/libs/immutable-interface.js
 /shared/libs/flow-interface.js
+/shared/constants/types/keybase-v1.js
```

Comment:
we don't necessarily control the data so lets not break ourselves if this doesn't pass link